architecture-building-systems/CityEnergyAnalyst
# 2117 weather as input file

This PR introduces a new tool `cea weather-helper` that can set the weather file for a scenario. The weather file is saved to `locator.get_weather_file()` which is located in `inputs/weather/weather.epw`.

I have tried to test as much as I can - but due to problems with the workflow tools, I couldn't run a _whole_ test. I think I got everything, though.

Test like this:

- `cea test`
- run `cea dashboard`
  - create new project
  - create new scenario (generate, select weather checkbox)
    - test once with setting the weather-helper settings to and once with default
  - check output scenario (`inputs/weather/weather.epw`)
    - should be the chosen weather location
  - run `weather-helper` tool to change the weather file, check `weather.epw` for new location
  - run `demand` etc. to see if they pick up the weather
  - delete the weather file, rerun demand - expect an error message

There are some scripts that I didn't change in the neural network and surrogate folders - they don't run anyway, and will be moved to legacy.
